device-types/Server Technology/C1W08HC-0ABA2BAC.yaml

```diff
@@ -0,0 +1,67 @@
+---
+manufacturer: Server Technology
+model: C1W08HC-0ABA2BAC
+slug: server-technology-c1w08hc-0aba2bac
+part_number: C1W08HC-0ABA2BAC
+u_height: 1
+is_full_depth: false
+airflow: passive
+comments: |
+  PRO1 Switched, Per-Inlet Power Sensing, Primary, 16A, 120-240V, (8) C13,
+  Dimensions: 45 x 437 x 178 mm (1.75 x 17.2 x 7.0")
+console-ports:
+  - name: ser
+    label: SER
+    type: rj-45
+  - name: th1
+    label: T/H1
+    type: rj-11
+  - name: th2
+    label: T/H2
+    type: rj-11
+console-server-ports:
+  - name: link
+    label: LINK
+    type: rj-12
```

Comment:
Is this port actually used to provide remote access to the local console of a connected downstream device? If not then it should not be modelled under `console-server-ports`

Reply:
This is technically used to daisy chain PDU's together to share a common management  ethernet port. As mentioned with the other comment, this just follows a previously accepted merge - I think the intention was to cover all physical ports.

Reply:
If that is the ports usage, then that is fine. Thanks for clarifying